packages/components/docs/localDev.md

```diff
@@ -181,24 +181,31 @@ update the `package.json` file and commit that change.  Then you can do a `git p
 Note that for some of our premium npm packages, we are using `npm` to do the build/publish. In those cases, you'll need
 to update the package.json file version number manually before running `npm publish`.
 
-## Merging changes into develop
+## Merging labkey-ui-components changes into develop
 
-1. Message the Frontend dev room chat about starting the pull request merge. This is to make sure two people aren't
-merging at the same time which might result in conflicting package version numbers.
 1. Do one final merge of the `develop` branch into your feature branch for `labkey-ui-components`.
-1. Run one final lint of your changes, `yarn run lint-branch-fix`, and review the changes applied.
-1. Update the `releaseNotes/labkey/components.md` file with what will be your release version number and release date.
-1. Run the commands to build, test, and publish: `yarn build`, `yarn test`, `yarn publish`.
-1. Push your final set of commits from `labkey-ui-components` to github so that TeamCity can do a final run of the jest tests.
+2. Run one final lint of your changes, `yarn run lint-branch-fix`, and review the changes applied.
+3. Update the `releaseNotes/labkey/components.md` file with what will be your release version number and release date.
+4. Run the commands to build and test, and publish: `yarn build`, `yarn test`, `yarn publish`.
+5. Push your final set of commits from `labkey-ui-components` to GitHub so that TeamCity can do a final run of the jest tests.
+6. Message the Frontend dev room chat about starting the pull request merge. This is to make sure two people aren't
```

Comment:
I believe this messaging should really come first, or at least before you update the release number and date and publish with that release version.  If you publish a release version before letting people know, someone else may decide to publish a next version as well, and that's the problem the messaging to humans is meant to avoid.  In tech-y terms, I think you want to acquire your lock on the releasing process earlier.

Reply:
Agreed - it should be before the publish. I meant to separate the 'yarn publish' from the build and test step, but failed to delete it during the copy/paste of the line.

I pushed a correction that has build/test as step 4, messaging as 6, and publishing as 9.

Does that seem appropriate to you? I'm trying to say, get your ducks in a row, let everyone know you're going to claim the version number, and then do it.